test/scenarios/verifier/.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+---
+repos:
+  - repo: git://github.com/pre-commit/pre-commit-hooks
```

Comment:
Is there some chance to vendor this locally? We have a lot of issues with Github network connectivity in the functional tests. They fail a bunch. It would be nice to avoid this.

Reply:
`pre-commit` by default caches it's repositories in `~/.cache/pre-commit`, should I add that to `.travis-ci.yml` would that be sufficient?

Reply:
And that directory is already in the `.travis-ci.yml`. The other option is I can just call out to `flake8` as an executable instead of pulling in the hooks repository.